apps/researcher/src/app/[locale]/communities/[slug]/page.tsx

```diff
@@ -69,70 +69,59 @@ export default async function CommunityPage({params}: Props) {
-            baseUrl="/communities"
-            className="flex items-center gap-1 no-underline"
-          >
-            <ChevronLeftIcon className="w-4 h-4 fill-neutral-500" />
-            {t('backButton')}
-          </ToFilteredListButton>
+      <div className="bg-consortiumGreen-300 text-consortiumBlue-800 relative z-20">
+        <div className="w-full px-4 sm:px-10 max-w-[1800px] mx-auto pt-10 flex justify-between">
+          <div>
+            <ToFilteredListButton
+              baseUrl="/communities"
+              className="flex items-center gap-1 no-underline"
+            >
+              <ChevronLeftIcon className="w-4 h-4 fill-consortiumBlue-800" />
+              {t('backButton')}
+            </ToFilteredListButton>
+          </div>
+          <div>
+            <Protect
+              communityId={community.id}
+              permission="org:sys_profile:manage"
+            >
+              <div className="w-full flex justify-end -mb-8">
+                <SlideOutButton
+                  id={slideOutEditFormId}
+                  className="p-1 sm:py-2 sm:px-3 text-sm rounded-full bg-consortiumBlue-800 text-consortiumGreen-300 transition flex items-center gap-1 hover:bg-consortiumBlue-700"
+                >
+                  <PencilSquareIcon className="w-5 h-5 fill-consortiumGreen-300" />
+                  {t('editButton')}
+                </SlideOutButton>
+              </div>
+            </Protect>
+          </div>
         </div>
-      </div>
-      <div className="flex flex-col md:flex-row h-full items-stretch grow content-stretch self-stretch gap-4 md:gap-16 w-full max-w-[1800px] mx-auto px-4 sm:px-10 mt-12">
-        <main className="w-full">
-          <Protect
-            communityId={community.id}
-            permission="org:sys_profile:manage"
-          >
-            <div className="w-full flex justify-end -mb-8">
-              <SlideOutButton
-                id={slideOutEditFormId}
-                className="p-1 sm:py-2 sm:px-3 rounded-full text-xs bg-neutral-200/50 hover:bg-neutral-300/50 text-neutral-800 transition flex items-center gap-1"
-              >
-                <PencilSquareIcon className="w-5 h-5 fill-neutral-700" />
-                {t('editButton')}
-              </SlideOutButton>
-            </div>
-          </Protect>
-          <div className="-mb-16 md:-mb-24 w-full flex justify-center">
-            <div className="w-32 h-32 lg:w-48 lg:h-48 rounded-full overflow-hidden relative">
+        <div className="w-full px-4 sm:px-10 max-w-[1800px] mx-auto pt-4 pb-10 md:pt-10 md:pb-16 xl:pt-16 xl:pb-20 flex flex-col lg:flex-row gap-10">
+          <SlideOutClosed id={slideOutEditFormId}>
+            <main className="w-full lg:w-2/3 xl:w-3/4">
+              <h1 className="text-2xl md:text-3xl lg:text-4xl 2xl:text-5xl max-w-5xl inline-flex gap-2">
+                <span className="font-normal">{t('title')}</span>
+                <span>{community.name}</span>
```

Comment:
Can you add `data-testid="community-name"` here? This is needed for the end-to-end tests